scripts/install.sh

```diff
@@ -208,6 +212,15 @@ function init() {
     exportKubeconfig
     KUBEADM_TOKEN_CA_HASH=$(cat /tmp/kubeadm-init | grep 'discovery-token-ca-cert-hash' | awk '{ print $2 }' | head -1)
 
+    if [ "$CIS_COMPLIANCE" == "1" ]; then
+        kubectl apply -f $kustomize_kubeadm_init/pod-security-policy-privileged.yaml
+        # patch 'PodSecurityPolicy' to kube-apiserver and wait for kube-apiserver to reconcile
+        old_admission_plugins='--enable-admission-plugins=NodeRestriction'
+        new_admission_plugins='--enable-admission-plugins=NodeRestriction,PodSecurityPolicy'
+        sed -i "s%$old_admission_plugins%$new_admission_plugins%g"  /etc/kubernetes/manifests/kube-apiserver.yaml
+        spinner_kubernetes_api_stable
```

Comment:
we still need a wait before starting this to avoid races, and a log to tell us what we're waiting for

Reply:
https://github.com/replicatedhq/kURL/blob/978744d11c332ecda957fedd15e2de86e9f9781a/scripts/common/kubernetes.sh#L289-L296

`spinner_kubernetes_api_stable` does that internally

Reply:
+ added logging
https://github.com/replicatedhq/kURL/blob/a83988476583d417acede5ebb6bda173a7a669a7/scripts/common/kubernetes.sh#L289-L297